Doc/Bib/statistics.html

```diff
@@ -0,0 +1,64 @@
+---
+title: GAP Bibliography Statistics
+layout: default
+---
+
+<table>
+<tr>
+  <td>
+    <b>Year</b>
+  </td>
+  <td>
+    <b>Number of citations</b>
+  </td>  
+</tr>
+{% for year in (1987 .. 2020) %}
+<tr>
+  <td>
+    {{ year }}
+  </td>
+  <td>
+    {{ site.data.bib_stats_year[year] }}
+  </td>  
+</tr>
+{% endfor %}
```

Comment:
It would be more natural and safer to simply iterate over all years defined in the statistics (UNTESTED but the general principle I've used often enough):
```suggestion
{% for stat in site.data.bib_stats_year %}
<tr>
  <td>
    {{ stat[0] }}
  </td>
  <td>
    {{ stat[1] }}
  </td>  
</tr>
{% endfor %}
```

Reply:
Thanks! I tried the generic approach which failed, now I understood what was wrong there.